build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 group 'io.coala.jetbrains'
-version '0.0.1'
+version '1.0'
```

Comment:
I don't think we should bump it to 1. Maybe `0.1`? It needs some testing from other contributors as well.

Reply:
I was thinking the same, this is more like alpha/beta atm. I'll go with `0.1`